twarep/electrifly
# Completed Model Update

## Checklist

## What I Did
- Changed around how the model takes data and made sure that the new flight activities are in the model selection. Plus they work.

## How I Did it?
- Just used python.

## How to Test?
- Run a bunch of tests on the ML model UI and make sure that the code is working as intended for the flights

### Relevant Links
Any tutorials or other resources used to help with this task. You can also cite the documentation.
